src/client/pages/Embark/index.tsx

```diff
@@ -121,7 +121,16 @@ const Embark: React.FunctionComponent<EmbarkProps> = (props) => {
         embarkPassageName: props.name,
       })
     }
-  }, [currentPassage.id])
+  }, [
+    currentPassage.id,
+    currentPassage.url,
+    currentPassage.api,
+    currentPassage.externalRedirect,
+    history,
+    state,
+    props.baseUrl,
+    props.name,
```

Comment:
Feel free to omit in this PR, would be nice to deconstruct these 🤗

Reply:
I agree with you but made the decision to not have those kind of changes in this PR, I think it makes it cleaner and easier to review if this is purely fixing linting stuff 😇

Reply:
Word! 👌